config/sync/block.block.blotlagtformularsearch_topicspage.yml

```diff
@@ -0,0 +1,29 @@
+uuid: 05952cdc-bc81-4b78-ba2b-976cee631623
+langcode: da
+status: true
+dependencies:
+  config:
+    - views.view.search_topics
+  module:
+    - system
+    - views
+  theme:
+    - itchefer
+id: blotlagtformularsearch_topicspage
```

Comment:
The id `blotlagtformularsearch_topicspage` is a weird mix of Danish and English. Should we do something about that?

Reply:
Was wondering why it was exported like that - can I just change an id without repercussions?